newt/vendor/mynewt.apache.org/newt/util/util.go

```diff
@@ -42,6 +42,7 @@ import (
 )
 
 var Verbosity int
+var PrintBuildCmds bool
```

Comment:
Cool, looks good.  What do you think about renaming `PrintBuildCmds` to `PrintShellCmds` (or similar)?  Other tools use the util library, but they don't build anything.